src/packages/engine-core/package.json

```diff
@@ -51,7 +51,7 @@
     "new-github-issue-url": "^0.2.1",
     "p-retry": "^4.2.0",
     "terminal-link": "^2.1.1",
-    "undici": "3.3.6"
+    "undici": "^4.0.0"
```

Comment:
I don't know why but pnpm also keeps the version 3.x.x which causes build of `cli` to fail if I remove `_http_common` from `esbuild.external`.

Reply:
also experiencing errors when  _http_common is removed from Webpack's externals